src/web/mjs/connectors/Baozimh.mjs

```diff
@@ -0,0 +1,47 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Baozimh extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'baozimh';
+        super.label = '包子漫書 (baozimh)';
+        this.tags = [ 'manga', 'webtoon', 'chinese' ];
+        this.url = 'https://www.baozimh.com/';
+    }
+	
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+		const data = await this.fetchDOM(request, '.comics-detail__title'); //title object
+        const id = uri.pathname;
+		let title = data[0].textContent.trim()
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
```

Comment:
Why not providing the list which is directly shown on the website https://www.baozimh.com?

Reply:
I didn't add it as I never used that feature in hakuneko and saw that a lot of other connectors also didn't have it.
I will try to add it when I fix the other things from your review

Reply:
I checked it out. they work with pagination 
https://www.baozimh.com/classify?page=1
is the url.
the problem is that they are a aggrigator and even when i was at page 41 in my browser it still went on, it has even more then 700 pages with 36 manhua's each. making 700 requests would not be good for the user experience even if the website doesn't block making so much request to the website.